FRENCH/src/ch19-05-advanced-functions-and-closures.md

```diff
@@ -207,9 +207,9 @@ compile to the same code, so use whichever style is clearer to you.
 Nous avons ici créé des instances de `Statut::Valeur` en utilisant chacune des
 valeurs `u32` présentes dans l'intervalle sur laquelle nous appelons `map` en
 utilisant la fonction d'initialisation de `Statut::Valeur`. Certaines personnes
-préfèrent ce style, et d'autres préfèrent utiliser des fermetures. Ces deux cas
-se compilent pour faire la même chose, donc vous pouvez utiliser le style qui
-vous conviens le plus.
+préfèrent ce style, et d'autres préfèrent utiliser des fermetures. Ces deux approches
+se compilent et produisent le même code, vous pouvez donc utiliser le style qui
```

Comment:
```suggestion
se compilent et produisent la même logique, vous pouvez donc utiliser le style qui
```

Reply:
ok

Reply:
I understand the same machine code here ... more 'code' than 'logique' for me, so.

Reply:
Oh, I think I misunderstood this sentence and by the way your translation follows the initial word of the English book, so we will keep your suggestion, thank you ! 😉